pyteal/errors.py

```diff
@@ -15,6 +15,17 @@ def __str__(self):
 TealInternalError.__module__ = "pyteal"
 
 
+class TealSeqError(Exception):
```

Comment:
It would be nice if this could inherit from `TealTypeError`, since it's a more specific type error